bittensor/_neuron/text/advanced_server/run.py

```diff
@@ -192,6 +192,10 @@ def time_check():
 
     nn = subtensor.neuron_for_pubkey(wallet.hotkey.ss58_address)
 
+    # --- last sync block 
+    last_sync_block = subtensor.get_current_block()
+    last_set_block = subtensor.get_current_block()
```

Comment:
Do we need to call subtensor twice here? why not just do `last_set_block = last_sync_block`?

Reply:
sounds good, I will change that.